audformat/core/table.py

```diff
@@ -1479,6 +1479,47 @@ def get(
 
         return result
 
+    def map_files(
+            self,
+            func: typing.Callable[[str], str],
+    ):
+        r"""Apply function to file names in table.
+
+        Relies on :meth:`pandas.Index.map`,
+        which can be slow.
+        If speed is crucial,
+        consider to change the index directly.
+        In the following example we prefix every file with a folder:
+
+        .. code-block:: python
+
+            root = '/root/'
+            if table.is_filewise:
+                table.df.index = root + table.df.index
+                table.df.index.name = audformat.define.IndexField.FILE
+            elif len(table.df.index) > 0:
+                table.df.index.set_levels(
+                    root + table.df.index.levels[0],
+                    audformat.define.IndexField.FILE,
+                    inplace=True,
+                )
```

Comment:
This example could be replaced by

```python
table.df.index = audformat.utils.expand_file_path(table.df.index, root)
```

Reply:
I would not change to a utility function here since the purpose is to give the user some hint how to optimize independent of the actual task.

Reply:
OK, then maybe change to an example that cannot be done by a utility function?

Reply:
I changed the example and moved it to `utils.map_file_path()`.